README.rst

```diff
@@ -50,6 +58,15 @@ can use full urls or relative API urls:
     amigocloud.get('/me')
     amigocloud.get('https://www.amigocloud.com/api/v1/me')
 
+For convenience, when using project tokens, urls are relative to the project's url (unless it starts with `/`):
+
+.. code:: python
+
+    # All three will do the same request:
+    amigocloud.get('datasets')
+    amigocloud.get('/users/123/projects/1234/datasets')
```

Comment:
Document what happens if you pass the wrong project url (Is AmigoCloudError thrown ?)